dask_ml/compose/_column_transformer.py

```diff
@@ -185,7 +185,13 @@ def _hstack(self, Xs):
                 warnings.filterwarnings("ignore", "Concatenating", UserWarning)
                 return dd.concat(Xs, axis="columns")
         elif da.Array in types:
-            return da.hstack(Xs)
+            # To allow compatibility with dask core 1.0.0, this is the
+            # definition of the dask.array.hstack inlined:
+            if all(x.ndim == 1 for x in Xs):
+                # tup, axis, allow_unknown_chunksizes
+                return da.concatenate(Xs, 0, True)
+            else:
+                return da.concatenate(Xs, 1, True)
```

Comment:
Can you use keywords for the 1 and True here?

Reply:
absolutely.